harvard-lts/CURIOSity
# Curiosity 12-8-2022

**Bring in changes from SoftServ -- 12/8/2022**
* * *


# What does this Pull Request do?
Brings over the latest main from softserv including:

- add psql client (part of migration)
- hide item embed widget https://github.com/orgs/harvard-lts/projects/4/views/1
- high priority design changes https://github.com/harvard-lts/CURIOSity/issues/248
- headings adjustments https://github.com/harvard-lts/CURIOSity/issues/86
- use oaipmh gem beta14 (part of design and art of solr harvest functions)

# How should this be tested?

Pull locally and make sure things come up and function -- all these have been tested before merge into softserv's main

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
